core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -44,6 +46,21 @@ public function parse()
 		return $strBuffer;
 	}
 
+	/**
+	 * Return a response object
+	 *
+	 * @param bool $blnForceCacheHeaders
```

Comment:
? :)

Reply:
Removed in 2b1d383e9cbf0bddc29705bda455c93c58878ea5. 😊